bin/methods/start.js

```diff
@@ -1,23 +1,33 @@
 module.exports = start
 const { writeFile } = require('fs')
-const path = require('path')
+const { resolve } = require('path')
 const log = require('log-cb')
 const { Client } = require('discord.js')
 
-function start (flags, plugins = []) {
+function start (flags) {
   // Load configuration file
   let config = {}
-  const configPath = path.resolve(process.cwd(), 'package.json')
+  const configPath = resolve(process.cwd(), 'package.json')
   try {
     config = require(configPath)
   } catch (e) {
     if (e.code !== 'MODULE_NOT_FOUND') return log()(e)
   }
 
+  // Set defaults
+  if (!config.loader) config.loader = 'cordlr-loader'
+  if (!config.plugins) config.plugins = []
+
   // Add command-line options to config
   const uneditedConfig = Object.assign({}, config)
   Object.assign(config, flags)
   delete config._
+  // add command-line plugins to config (if they weren't there yet)
+  for (const plugin of flags._) {
```

Comment:
This is confusing IMO.  Loaders shouldn't need to handle command-line specific things.

We should handle `flags._` and add it to `flags.plugins` before it reaches the loader.